resources/framebuffer.lisp

```diff
@@ -80,5 +80,12 @@
 (defmethod activate ((framebuffer framebuffer))
   (gl:bind-framebuffer :framebuffer (gl-name framebuffer))
   (gl:viewport 0 0 (width framebuffer) (height framebuffer))
-  ;; FIXME: Figure out which to clear depending on framebuffer attachments
+  ;; FIXME: Figure out which to clearq depending on framebuffer attachments
```

Comment:
A typo?

Reply:
Yeah, a stray q from trying to close buffers, most likely.